src/components/Tree/styled/spinner.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
+import Spinner from '../../Spinner';
+
+const BrandSpinner = attachThemeAttrs(styled(Spinner))`
+    color: ${props => props.palette.brand.main};
+    margin-top: 18px;
+    margin-left: 8px;
```

Comment:
I think we should avoid using these margins here, instead, we can add a container with 
`position: relative;
 display: flex;
 justify-content: center;
 align-items: center;`